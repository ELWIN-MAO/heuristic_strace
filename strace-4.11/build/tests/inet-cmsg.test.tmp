
mym_waitpid=101912

mym_waitpid=101912

mym_waitpid=101912

syscall_enter_mym_scno=59  sys_name=execve

mym_waitpid=101912

mym_waitpid=101912

syscall_exit_mym_scno=59  sys_name=execve

mym_waitpid=101912

syscall_enter_mym_scno=12  sys_name=brk

mym_waitpid=101912

syscall_exit_mym_scno=12  sys_name=brk

mym_waitpid=101912

syscall_enter_mym_scno=21  sys_name=access

mym_waitpid=101912

syscall_exit_mym_scno=21  sys_name=access

mym_waitpid=101912

syscall_enter_mym_scno=9  sys_name=mmap

mym_waitpid=101912

syscall_exit_mym_scno=9  sys_name=mmap

mym_waitpid=101912

syscall_enter_mym_scno=21  sys_name=access

mym_waitpid=101912

syscall_exit_mym_scno=21  sys_name=access

mym_waitpid=101912

syscall_enter_mym_scno=2  sys_name=open

mym_waitpid=101912

syscall_exit_mym_scno=2  sys_name=open

mym_waitpid=101912

syscall_enter_mym_scno=5  sys_name=fstat

mym_waitpid=101912

syscall_exit_mym_scno=5  sys_name=fstat

mym_waitpid=101912

syscall_enter_mym_scno=9  sys_name=mmap

mym_waitpid=101912

syscall_exit_mym_scno=9  sys_name=mmap

mym_waitpid=101912

syscall_enter_mym_scno=3  sys_name=close

mym_waitpid=101912

syscall_exit_mym_scno=3  sys_name=close

mym_waitpid=101912

syscall_enter_mym_scno=21  sys_name=access

mym_waitpid=101912

syscall_exit_mym_scno=21  sys_name=access

mym_waitpid=101912

syscall_enter_mym_scno=2  sys_name=open

mym_waitpid=101912

syscall_exit_mym_scno=2  sys_name=open

mym_waitpid=101912

syscall_enter_mym_scno=0  sys_name=read

mym_waitpid=101912

syscall_exit_mym_scno=0  sys_name=read

mym_waitpid=101912

syscall_enter_mym_scno=5  sys_name=fstat

mym_waitpid=101912

syscall_exit_mym_scno=5  sys_name=fstat

mym_waitpid=101912

syscall_enter_mym_scno=9  sys_name=mmap

mym_waitpid=101912

syscall_exit_mym_scno=9  sys_name=mmap

mym_waitpid=101912

syscall_enter_mym_scno=10  sys_name=mprotect

mym_waitpid=101912

syscall_exit_mym_scno=10  sys_name=mprotect

mym_waitpid=101912

syscall_enter_mym_scno=9  sys_name=mmap

mym_waitpid=101912

syscall_exit_mym_scno=9  sys_name=mmap

mym_waitpid=101912

syscall_enter_mym_scno=9  sys_name=mmap

mym_waitpid=101912

syscall_exit_mym_scno=9  sys_name=mmap

mym_waitpid=101912

syscall_enter_mym_scno=3  sys_name=close

mym_waitpid=101912

syscall_exit_mym_scno=3  sys_name=close

mym_waitpid=101912

syscall_enter_mym_scno=9  sys_name=mmap

mym_waitpid=101912

syscall_exit_mym_scno=9  sys_name=mmap

mym_waitpid=101912

syscall_enter_mym_scno=9  sys_name=mmap

mym_waitpid=101912

syscall_exit_mym_scno=9  sys_name=mmap

mym_waitpid=101912

syscall_enter_mym_scno=9  sys_name=mmap

mym_waitpid=101912

syscall_exit_mym_scno=9  sys_name=mmap

mym_waitpid=101912

syscall_enter_mym_scno=158  sys_name=arch_prctl

mym_waitpid=101912

syscall_exit_mym_scno=158  sys_name=arch_prctl

mym_waitpid=101912

syscall_enter_mym_scno=10  sys_name=mprotect

mym_waitpid=101912

syscall_exit_mym_scno=10  sys_name=mprotect

mym_waitpid=101912

syscall_enter_mym_scno=10  sys_name=mprotect

mym_waitpid=101912

syscall_exit_mym_scno=10  sys_name=mprotect

mym_waitpid=101912

syscall_enter_mym_scno=10  sys_name=mprotect

mym_waitpid=101912

syscall_exit_mym_scno=10  sys_name=mprotect

mym_waitpid=101912

syscall_enter_mym_scno=11  sys_name=munmap

mym_waitpid=101912

syscall_exit_mym_scno=11  sys_name=munmap

mym_waitpid=101912

syscall_enter_mym_scno=2  sys_name=open

mym_waitpid=101912

syscall_exit_mym_scno=2  sys_name=open

mym_waitpid=101912

syscall_enter_mym_scno=3  sys_name=close

mym_waitpid=101912

syscall_exit_mym_scno=3  sys_name=close

mym_waitpid=101912

syscall_enter_mym_scno=3  sys_name=close

mym_waitpid=101912

syscall_exit_mym_scno=3  sys_name=close

mym_waitpid=101912

syscall_enter_mym_scno=41  sys_name=socket

mym_waitpid=101912

syscall_exit_mym_scno=41  sys_name=socket

mym_waitpid=101912

syscall_enter_mym_scno=49  sys_name=bind

mym_waitpid=101912

syscall_exit_mym_scno=49  sys_name=bind

mym_waitpid=101912

syscall_enter_mym_scno=51  sys_name=getsockname

mym_waitpid=101912

syscall_exit_mym_scno=51  sys_name=getsockname

mym_waitpid=101912

syscall_enter_mym_scno=41  sys_name=socket

mym_waitpid=101912

syscall_exit_mym_scno=41  sys_name=socket

mym_waitpid=101912

syscall_enter_mym_scno=42  sys_name=connect

mym_waitpid=101912

syscall_exit_mym_scno=42  sys_name=connect

mym_waitpid=101912

syscall_enter_mym_scno=54  sys_name=setsockopt

mym_waitpid=101912

syscall_exit_mym_scno=54  sys_name=setsockopt

mym_waitpid=101912

syscall_enter_mym_scno=54  sys_name=setsockopt

mym_waitpid=101912

syscall_exit_mym_scno=54  sys_name=setsockopt

mym_waitpid=101912

syscall_enter_mym_scno=54  sys_name=setsockopt

mym_waitpid=101912

syscall_exit_mym_scno=54  sys_name=setsockopt

mym_waitpid=101912

syscall_enter_mym_scno=54  sys_name=setsockopt

mym_waitpid=101912

syscall_exit_mym_scno=54  sys_name=setsockopt

mym_waitpid=101912

syscall_enter_mym_scno=54  sys_name=setsockopt

mym_waitpid=101912

syscall_exit_mym_scno=54  sys_name=setsockopt

mym_waitpid=101912

syscall_enter_mym_scno=54  sys_name=setsockopt

mym_waitpid=101912

syscall_exit_mym_scno=54  sys_name=setsockopt

mym_waitpid=101912

syscall_enter_mym_scno=54  sys_name=setsockopt

mym_waitpid=101912

syscall_exit_mym_scno=54  sys_name=setsockopt

mym_waitpid=101912

syscall_enter_mym_scno=44  sys_name=sendto

mym_waitpid=101912

syscall_exit_mym_scno=44  sys_name=sendto

mym_waitpid=101912

syscall_enter_mym_scno=3  sys_name=close

mym_waitpid=101912

syscall_exit_mym_scno=3  sys_name=close

mym_waitpid=101912

syscall_enter_mym_scno=47  sys_name=recvmsg
recvmsg(0, 
mym_waitpid=101912

syscall_exit_mym_scno=47  sys_name=recvmsg
{msg_name(16)={sa_family=AF_INET, sin_port=htons(53182), sin_addr=inet_addr("127.0.0.1")}, msg_iov(1)=[{"data", 4}], msg_controllen=152, [{cmsg_len=28, cmsg_level=SOL_IP, cmsg_type=IP_PKTINFO, {ipi_ifindex=if_nametoindex("lo"), ipi_spec_dst=inet_addr("127.0.0.1"), ipi_addr=inet_addr("127.0.0.1")}}, {cmsg_len=20, cmsg_level=SOL_IP, cmsg_type=IP_TTL, {ttl=64}}, {cmsg_len=17, cmsg_level=SOL_IP, cmsg_type=IP_TOS, {tos=0}}, {cmsg_len=20, cmsg_level=SOL_IP, cmsg_type=IP_RECVOPTS, {opts=0x01000000}}, {cmsg_len=16, cmsg_level=SOL_IP, cmsg_type=IP_RETOPTS}, {cmsg_len=32, cmsg_level=SOL_IP, cmsg_type=IP_ORIGDSTADDR, {sa_family=AF_INET, sin_port=htons(48626), sin_addr=inet_addr("127.0.0.1")}}], msg_flags=0}, 0
mym_scall_rval=0x4
) = 4

mym_waitpid=101912

syscall_enter_mym_scno=3  sys_name=close

mym_waitpid=101912

syscall_exit_mym_scno=3  sys_name=close

mym_waitpid=101912

syscall_enter_mym_scno=5  sys_name=fstat

mym_waitpid=101912

syscall_exit_mym_scno=5  sys_name=fstat

mym_waitpid=101912

syscall_enter_mym_scno=12  sys_name=brk

mym_waitpid=101912

syscall_exit_mym_scno=12  sys_name=brk

mym_waitpid=101912

syscall_enter_mym_scno=12  sys_name=brk

mym_waitpid=101912

syscall_exit_mym_scno=12  sys_name=brk

mym_waitpid=101912

syscall_enter_mym_scno=1  sys_name=write

mym_waitpid=101912

syscall_exit_mym_scno=1  sys_name=write

mym_waitpid=101912

syscall_enter_mym_scno=231  sys_name=exit_group

mym_waitpid=101912
+++ exited with 0 +++
