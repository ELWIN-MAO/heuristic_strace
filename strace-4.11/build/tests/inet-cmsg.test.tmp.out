recvmsg(0, {msg_name(16)={sa_family=AF_INET, sin_port=htons(53182), sin_addr=inet_addr("127.0.0.1")}, msg_iov(1)=[{"data", 4}], msg_controllen=152, [{cmsg_len=28, cmsg_level=SOL_IP, cmsg_type=IP_PKTINFO, {ipi_ifindex=if_nametoindex("lo"), ipi_spec_dst=inet_addr("127.0.0.1"), ipi_addr=inet_addr("127.0.0.1")}}, {cmsg_len=20, cmsg_level=SOL_IP, cmsg_type=IP_TTL, {ttl=64}}, {cmsg_len=17, cmsg_level=SOL_IP, cmsg_type=IP_TOS, {tos=0}}, {cmsg_len=20, cmsg_level=SOL_IP, cmsg_type=IP_RECVOPTS, {opts=0x01000000}}, {cmsg_len=16, cmsg_level=SOL_IP, cmsg_type=IP_RETOPTS}, {cmsg_len=32, cmsg_level=SOL_IP, cmsg_type=IP_ORIGDSTADDR, {sa_family=AF_INET, sin_port=htons(48626), sin_addr=inet_addr("127.0.0.1")}}], msg_flags=0}, 0) = 4
+++ exited with 0 +++
lo IP Address 127.0.0.1
ens33 IP Address 192.168.159.137
IP Address 127.0.0.1
IP Address 192.168.159.137
